lib/manageiq/environment.rb

```diff
@@ -54,7 +54,9 @@ def self.update_ui_thread
     end
 
     def self.install_bundler(root = APP_ROOT)
-      system!("gem install bundler -v '#{bundler_version}' --conservative") unless ENV["GITHUB_ACTIONS"]
+      if !system("which bundle", [:out, :err] => "/dev/null") || !system("bundle info bundler", [:out, :err] => "/dev/null")
```

Comment:
(!A || !B) == (A && B)

```suggestion
      if system("which bundle", [:out, :err] => "/dev/null") && system("bundle info bundler", [:out, :err] => "/dev/null")
```

Reply:
LOL. 🤣

Reply:
I changed the logic to use a guard clause to keep the logic "positive".

Reply:
Done